src/Http/ServerRequest.php

```diff
@@ -799,11 +799,8 @@ protected function _is($type, $args)
         if (isset($detect['accept']) && $this->_acceptHeaderDetector($detect)) {
             return true;
         }
-        if (isset($detect['param']) && $this->_paramDetector($detect)) {
-            return true;
-        }
```

Comment:
The early return is a better pattern in this case, easier to grasp and thus less mistakes when modifying.